cpa/parsing.go

```diff
@@ -167,8 +168,21 @@ func parseBundle(bundle map[string]string, rules ...LintRule) (*Policy, error) {
 		}
 	}
 
-	compiler := ast.NewCompiler()
-	compiler.WithStrict(true)
+	capabilities := ast.CapabilitiesForThisVersion()
+	capabilities.AllowNet = []string{}
+
+	disallowedBuiltins := []string{"http.send", "net.lookup_ip_addr"}
```

Comment:
While we can of course add to this later, how confident are we that these are the only ones we need to block for the time being?

Reply:
Fairly confident. There are two package builtins in in OPA that deal with networking.

net and http.

http.send is the only http builtin

there are a couple in net but lookup is the only that makes networking requests. regardless net.lookup is not dangerous, but its should be removed.